iOS/Collection View Management /CollectionViewDataSource.swift

```diff
@@ -331,7 +353,7 @@ extension CollectionViewDataSource {
         // so we don't preload any cells undesired cells when the snapshop is applied.
         if animationCycle.scrollToEnd {
             collectionView.contentOffset = CGPoint(x: 999_999_999_999,
-                                                   y: 999_999_999_999)
+                                                   y: 0)
```

Comment:
Changing the y value doesn't work for horizontal layouts. An improvement would be to make this work for both horizontal and vertical layouts.